composer.json

```diff
@@ -50,14 +50,14 @@
     "mockery/mockery": "^1.4",
     "nunomaduro/phpinsights": "^2.0",
     "nyholm/psr7": "^1.4",
-    "pestphp/pest": "^1.0",
-    "pestphp/pest-plugin-parallel": "^0.2.0",
+    "pestphp/pest": "^1.18",
```

Comment:
Ensure we're seeding earliest compatible versions of Pest and Psalm for the new 8.1 tests.